src/assets/Keycloak/Keycloak.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Introduction to Keycloak
+authorName: Amrita Shrestha
+authorAvatar: https://avatars.githubusercontent.com/u/54478846?s=100&v=4
+authorLink: https://github.com/amrita-shrestha
+createdAt: May 13, 2024
+tags: Keycloak, Identity Access Management
+banner: https://blog.jankaritech.com/src/assets/Keycloak/images/Keycloak/Keycloak.png
+---
+
+## Keycloak
+[Keycloak](https://github.com/keycloak/keycloak) is an open-source project created by RedHat for Single Sign-On. It provides an `Identity and Access Management` (IAM) solution designed to secure application services.
+Additionally, it enables users to authenticate through various identity providers and use fine-grained permissions for regulating access to Software as a Service (SaaS) applications.
+
+It facilitates the creation of a user database with customizable roles and groups, offering functionalities such as user management, registration, and password policy enforcement.
+This makes it a comprehensive Identity and Access Management solution for contemporary applications. Keycloak API enables the integration of all these features into your application
+without the need for additional coding.
+
+## Keycloak Features
+![Keycloak features](/src/assets/Keycloak/images/Keycloak/Keycloak-features.png)
+
+1. Identity Broker:
+   Identity Broker acts as a middleman, linking various service providers with different identity providers. It establishes trust with external identity providers to utilize their identities for accessing internal services offered by service providers.
+   Imagine a textile company, which has two Keycloak setups: one for customer and another for internal employees. If you want your employees to access external services without creating new accounts on customer Keycloak, you can use the customer Keycloak
```

Comment:
```suggestion
   Imagine a textile company, which has two Keycloak setups: one for customers and another for internal employees. If you want your employees to access external services without creating new accounts on the customer Keycloak, you can use the customer Keycloak
```

Reply:
It's a general example, do we need to go that deep? Blog is about Keycloak so may be avoiding non-IT related thing will be good otherwise blog will be full of less useful information

Reply:
If it's a general example, then why specifying that this is a textile company... I think this just adds confusion to the reader